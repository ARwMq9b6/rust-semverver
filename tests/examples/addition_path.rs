pub mod old {
    pub mod a {
        pub struct Abc;
    }
}

pub mod new {
    pub mod a {
        pub struct Abc;
    }

    pub mod b {
        pub use new::a::*;
    }

    pub mod c {
        pub use new::a::Abc;
    }
}
