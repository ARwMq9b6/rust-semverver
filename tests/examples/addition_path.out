max: TechnicallyBreaking
  Addition: ::b
  Addition: ::b::Abc
  Addition: ::c
  Addition: ::c::Abc
